cuckoo/reporting/maecreport.py

```diff
@@ -673,45 +716,53 @@ def addCapabilities(self):
                 if name_chunk in label_mappings:
                     if 'labels' not in self.primaryInstance:
                         self.primaryInstance['labels'] = []
-                        self.primaryInstance['labels'].append(label_mappings[name_chunk])
+                        self.primaryInstance['labels'].append(
+                            label_mappings[name_chunk])
                     else:
-                        self.primaryInstance['labels'].append(label_mappings[name_chunk])
+                        self.primaryInstance['labels'].append(
+                            label_mappings[name_chunk])
                 if name_chunk in capability_mappings:
                     # We're dealing with a "top-level" Capability
                     if len(capability_mappings[name_chunk].keys()) == 1:
                         capabilities.append(capability_mappings[name_chunk])
                     # We're dealing with a "nested" Capability
                     elif len(capability_mappings[name_chunk].keys()) == 2:
-                        # Make sure the 'parent' Capability isn't already defined
+                        # Make sure the 'parent' Capability isn't already
+                        # defined
                         capability = capability_mappings[name_chunk]
                         for cap in capabilities:
                             if cap['name'] == capability['name']:
                                 capability = cap
                         if 'refined_capabilities' not in capability:
-                            capability['refined_capabilities'] = capability_mappings[name_chunk]['refined_capabilities']
+                            capability[
+                                'refined_capabilities'] = capability_mappings[
+                                    name_chunk][
+                                        'refined_capabilities']
                         else:
-                            names = [x['name'] for x in capability['refined_capabilities']]
-                            if capability_mappings[name_chunk]['refined_capabilities'][0]['name'] not in names:
-                                capability['refined_capabilities'].append(capability_mappings[name_chunk]['refined_capabilities'][0])
+                            names = [x['name'] for x in capability
+                                     ['refined_capabilities']]
+                            if capability_mappings \
```

Comment:
In general, I don't think we need to be too strict about sticking to an 80-character limit for line length. In our other libraries our style checker uses a 160-character limit. Sometimes making lines fit in 80 characters reduces readability.

Another way to shorten long lines of code is to use intermediate variables. Here, for example:
```
refined_capability = capability_mappings[name_chunk]['refined_capabilities'][0]
if refined_capability['name'] not in names:
    capability['refined_capabilities'].append(refined_capability)
```

Another thing we could do is pull some of this code out to its own function, reducing how far it's indented.

Putting function arguments on their own lines, as you've done elsewhere, is also good.